bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -614,7 +614,19 @@ public void safeRun() {
         }
     }
 
-    void checkForFaultyBookies() {
+    void checkForFaultyBookies()  {
+        boolean isEnable = true;
+        try {
+            isEnable = metadataDriver.isEnableHealthCheck().get();
+        } catch (Exception e) {
+            LOG.error("call method isEnableHealthCheck failed!", e);
```

Comment:
in case of error, we could explain in the log that the value is treated as false

Reply:
call method isEnableHealthCheck failed，isEnable is treated   as true.

Reply:
yes, from a user/admin side I'd like to read something like
`can't determine if the bookie health checker is enabled, falling back to the default behavior (checker enabled)` or similar

Reply:
OK,I will fixed!